### PR TITLE
First blog: GSoC

### PR DESCRIPTION
**Pull request by xtanion**
My Journey to Google Summer of Code 2022

### CODE REVIEW

File: docs/source/posts/2022/2022-05-24-my-journey-to-gsoc-2022-shivam.rst [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+    :category:: gsoc
+
+
+About Myself
+~~~~~~~~~~~~
+
+Hi! I'm Shivam, currently pursuing my bachelor's (expected 2024) in Production and Industrial engineering from the Indian Institute of Technology (IIT) Roorkee.
+
+I was introduced to the C programming language through the Introduction to programming course in my first year of college. I always liked computers and I was overwhelmed by printing "hello world" in the terminal back then. Computer programming got me hooked up, and I knew this is what I wanted to do for the rest of my life. I learned python on my own and made a lot of projects. I participated in a lot of hackathons, won some of them, and lose a lot.
+In my second semester, I got to know about open-source, git, and GitHub. I came across some computer graphics enthusiasts in my college, they told me that they created a 2D game engine on their own using OpenGL. This gave me enough motivation to learn OpenGL and basic computer graphics.
```
Comment (ganimtron-10) [inline docs/source/posts/2022/2022-05-24-my-journey-to-gsoc-2022-shivam.rst:16]: Extra comma(,) before the and.

**Review by ganimtron-10 — COMMENTED** [review]
Hey @xtanion ,
Below I have added a typo issue that I found in your blog.
PTAL.
Thanks!

File: docs/source/posts/2022/2022-05-24-my-journey-to-gsoc-2022-shivam.rst [hunk 2]
```diff
@@ -0,0 +1,53 @@
+My Journey to GSoC 2022
+===============
```
Comment (skoudoro) [inline docs/source/posts/2022/2022-05-24-my-journey-to-gsoc-2022-shivam.rst:2]: too short, should be the same size as the title. the doc generation creates multiple warnings.

File: docs/source/posts/2022/2022-05-24-my-journey-to-gsoc-2022-shivam.rst [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+
+..  post:: May 24 2022
+    :author:: Shivam Sahu
+    :tags:: google
+    :category:: gsoc
+
+
+About Myself
+~~~~~~~~~~~~~~~~~~~~~~~~~
```
Comment (skoudoro) [inline docs/source/posts/2022/2022-05-24-my-journey-to-gsoc-2022-shivam.rst:11]: too long, should be the same size as the title. the doc generation creates multiple warnings.

File: docs/source/posts/2022/2022-05-24-my-journey-to-gsoc-2022-shivam.rst [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+
+1. [https://github.com/fury-gl/fury/pull/520]
+2. [https://github.com/fury-gl/fury/pull/525]
+3. [https://github.com/fury-gl/fury/pull/533]
+4. [https://github.com/fury-gl/fury/pull/547]
+5. [https://github.com/fury-gl/fury/pull/556]
+6. [https://github.com/fury-gl/fury/pull/559]
+
+The Day
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```
Comment (skoudoro) [inline docs/source/posts/2022/2022-05-24-my-journey-to-gsoc-2022-shivam.rst:48]: too short, should be the same size as the title. the doc generation creates multiple warnings.

**Review by skoudoro — COMMENTED** [review]
Hi @xtanion,

Thank you for this blog post. Have you tried to generate your blog post? if yes, can you send a screenshot?

**Comment by skoudoro** [comment]
Thank you for the review @m-agour, @ganimtron-10. Please, make sure that you succeed to generate the blog post during the review which is not currently the case. Thanks

File: docs/source/posts/2022/2022-05-24-my-journey-to-gsoc-2022-shivam.rst (hunk 2 above)
Comment (xtanion) [reply docs/source/posts/2022/2022-05-24-my-journey-to-gsoc-2022-shivam.rst:2]: Hi @skoudoro , I tried generating the blog post. I got a warning while generating the doc
`/home/shivam/fury/docs/source/posts/2022/2022-05-24-my-journey-to-gsoc-2022-shivam.rst: WARNING: document isn't included in any toctree` . But the generated doc was fine.
Screenshot from 2022-05-28 09-46-04

**Review by skoudoro — APPROVED** [review]
LGTM! thanks! merging

**Pull request merged**